lib/bundle/checker.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Bundle
+  module Checker
+    module_function
+
+    NO_ACTION = [].freeze
+
+    def action_required_for(formula)
+      [formula]
```

Comment:
How is this an action?

Reply:
See https://github.com/Homebrew/homebrew-bundle/pull/327/review_comment/create#discussion_r209120013

Reply:
That link points to this PR?